From: Richard Acayan <mailingradian@gmail.com>
Date: Tue, 8 Oct 2024 19:07:43 -0400
Subject: [PATCH] arm64: dts: qcom: sdm670-google: specify acdb path

The Audio Calibration Database contains a length-prefixed array of
topologies, which can enable call audio on more VoLTE calls. Device
firmware has its own directory to find firmware. Specify the path to the
ACDB so it can be added to the firmware tree.

Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi b/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
index 82a8e6d..44c94f9 100644
--- a/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
@@ -1049,6 +1049,10 @@ dai@3 {
 	};
 };
 
+&q6core {
+	qcom,acdb-name = "qcom/sdm670/sargo/Global_cal.acdb";
+};
+
 &sdhc_1 {
 	supports-cqe;
 	mmc-hs200-1_8v;
